components/button/index.tsx

```diff
@@ -6,7 +6,7 @@ export default function Button(
   return (
     <button
       {...props}
-      className="inline-flex justify-center py-2 px-11 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-gray-700 hover:bg-gray-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-700">
+      className="inline-flex justify-center py-2 px-11 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-gray-700 mr-2 hover:bg-gray-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-700">
```

Comment:
I think adding mr-2 class to the general button component will cause styling issue when we have a requirement to put the button on the rightmost side of the page

Reply:
updated and changed to flex class